sites/dentaleconomics/server/templates/website-section/editorial-advisory-board.marko

```diff
@@ -0,0 +1,59 @@
+import { getAsObject } from '@base-cms/object-path';
+import contentListFragment from '@endeavorb2b/base-website-common/api/fragments/content-list';
+
+$ const { site } = out.global;
+$ const section = getAsObject(data, 'section');
+$ const query = {
+  sectionAlias: section.alias,
+  limit: 100,
+  queryFragment: contentListFragment,
+};
+
+<theme-pennwell-website-section-layout section=section>
+  <@head>
+    <cms-ad-gam-head />
+    <script src="https://www.google.com/recaptcha/api.js?onload=vueRecaptchaApiLoaded&render=explicit" async defer></script>
```

Comment:
I'm assuming this is supposed to mimic a contact us page?

Reply:
Per Brandon: "Create it as a website section, largely cloning the Contact Us page layout, and have them schedule each individual (Contact content type) to the page."

I scheduled as many contacts to the pages as that had existing, although a majority of the people listed on the old site are not available in the contacts to schedule.

I'm assuming the brand will have revision requests for the page once they are able to see it, which will differentiate it from the layout/design of the contact us page.

Reply:
Just wanted to double-check that this was the same thing.